.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"semi": false,
+	"singleQuote": true,
+	"bracketSpacing": true,
+	"tabWidth": 4,
+	"useTabs": true,
+	"trailingComma": "es5"
+}

```

Comment:
Do you need this file?

Reply:
Hi, I use this file because I have the prettier vsc plugin and it conflicts with eslin. With the file it worked ok eslint. I don't know how else to configure it and make it work.

Reply:
Understood, its okay, i was wondering if it was necessary